### PR TITLE
Correct Flyway error

### PR DESCRIPTION
**Pull request by mmosix**
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.flywaydb#flyway-sbt;3.1: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Some unresolved dependencies have extra attributes.  Check that these dependencies exist with the requested attributes.
[warn] 		org.flywaydb:flyway-sbt:3.1 (scalaVersion=2.10, sbtVersion=0.13)
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		org.flywaydb:flyway-sbt:3.1 (scalaVersion=2.10, sbtVersion=0.13) (/Users/moze/Desktop/dev/server/project/plugins.sbt#L15-16)
[warn] 		  +- default:server-build:0.1-SNAPSHOT (scalaVersion=2.10, sbtVersion=0.13)
sbt.ResolveException: unresolved dependency: org.flywaydb#flyway-sbt;3.1: not found
	at sbt.IvyActions$.sbt$IvyActions$$resolve(IvyActions.scala:313)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:191)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:168)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:156)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:156)
	at sbt.IvySbt$$anonfun$withIvy$1.apply(Ivy.scala:133)
	at sbt.IvySbt.sbt$IvySbt$$action$1(Ivy.scala:57)
	at sbt.IvySbt$$anon$4.call(Ivy.scala:65)
	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:93)
	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:78)
	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:97)
	at xsbt.boot.Using$.withResource(Using.scala:10)
	at xsbt.boot.Using$.apply(Using.scala:9)
	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:58)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:48)
	at xsbt.boot.Locks$.apply0(Locks.scala:31)
	at xsbt.boot.Locks$.apply(Locks.scala:28)
	at sbt.IvySbt.withDefaultLogger(Ivy.scala:65)
	at sbt.IvySbt.withIvy(Ivy.scala:128)
	at sbt.IvySbt.withIvy(Ivy.scala:125)
	at sbt.IvySbt$Module.withModule(Ivy.scala:156)
	at sbt.IvyActions$.updateEither(IvyActions.scala:168)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1439)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1435)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$90.apply(Defaults.scala:1470)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$90.apply(Defaults.scala:1468)
	at sbt.Tracked$$anonfun$lastOutput$1.apply(Tracked.scala:37)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1473)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1467)
	at sbt.Tracked$$anonfun$inputChanged$1.apply(Tracked.scala:60)
	at sbt.Classpaths$.cachedUpdate(Defaults.scala:1490)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1417)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1369)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[error] (*:update) sbt.ResolveException: unresolved dependency: org.flywaydb#flyway-sbt;3.1: not found

### CODE REVIEW

File: actor-server/project/plugins.sbt [hunk 1]
```diff
@@ -12,9 +12,9 @@ addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.7.5")
 
 addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
 
-addSbtPlugin("org.flywaydb" % "flyway-sbt" % "3.1")
+addSbtPlugin("org.flywaydb" % "flyway-sbt" % "4.0")
 
-addSbtPlugin("im.actor" %% "sbt-actor-api" % "0.7.9")
+addSbtPlugin("im.actor" %% "sbt-actor-api" % "0.7.8")
```
Comment (rockjam) [inline actor-server/project/plugins.sbt]: @mmosix why you downgraded this?

File: actor-server/project/plugins.sbt [hunk 2]
```diff
@@ -35,5 +35,5 @@ addSbtPlugin("im.actor" % "actor-sbt-houserules" % "0.1.9")
 addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.3.8")
 
 libraryDependencies ++= Seq(
-  "com.github.os72" % "protoc-jar" % "3.0.0-b2.1"
+  "com.github.os72" % "protoc-jar" % "3.0.0-b2"
```
Comment (rockjam) [inline actor-server/project/plugins.sbt]: @mmosix same question as on L:17

**Comment by rockjam** [comment]
@mmosix I also can't reproduce dependency resolution error. Can you try to checkout lastest master branch of actor-platform, and make a clean build?

**Comment by mmosix** [comment]
i just corrected the file, i didn't take note of the downgrade.

**Comment by mmosix** [comment]
> @mmosix I also can't reproduce dependency resolution error. Can you try to checkout lastest master branch of actor-platform, and make a clean build?

i should make a clean build of what?
Explain more so i would understand, thank you.

**Comment by rockjam** [comment]
```
git checkout origin/master -t
sbt clean
sbt update
sbt compile
```

**Comment by rockjam** [comment]
@mmosix yes, what I meant - is to compile server like at first time

**Comment by rockjam** [comment]
@mmosix any progress here?

**Comment by mmosix** [comment]
@rockjam i build the server with my own forked repo. i have the file edited there. i just hope it wouldn't affect someone else if its not corrected from the master.

**Comment by rockjam** [comment]
@mmosix I wasn't able to reproduce it. It could be problem with flyway artifactory for limited time, or it's not available from certain regions

**Pull request closed without merging**